### DISCUSSION TITLE
[Feature] Possible to give option to attach traces ?

### DISCUSSION THREAD

**Discussion by shortcut2yp** [ortoniKC/ortoni-report#25] [Ideas]
As per our last conversation , I guess you already have a code to check if traces are generated but we are not adding them in the report yet. Possible to give a config option in next release to enable attaching the traces in the report. In your latest 2.0.3 I guess you are trying to host a server and then render traces ( guessing as traces need http / https ) but  if someone is already using jenkins / github actions / aws since the report will already be rendered on jenkins / github actions / s3 URL , probably opening traces also might work.  #JustThinkingLoud
As default playwright report traces works for me on Jenkins in this way.

**Comment by ortoniKC** [comment]
You have done the research, great. 
I'm trying my best.

**Reply by ortoniKC** [reply]
Trace is available now and it works in the local environment, I'm not sure if it works on the ci.
